WordPress/gutenberg
# Use WordPress URL module instead of Gutenberg's own function

**Testing instructions**

- Check that the preview button works as expected (correct URL)
- Check that deleting a post redirect to the post list correctly.
